test/TEST-04-FULL-SYSTEMD/test-init.sh

```diff
@@ -37,5 +37,4 @@ if getargbool 0 rd.shell; then
     setsid $CTTY sh -i
 fi
 echo "Powering down."
-systemctl --no-block poweroff
-exit 0
+poweroff -f
```

Comment:
Why are we changing the commands here?

Reply:
To align with all the other tests - e.g. https://github.com/dracut-ng/dracut-ng/blob/main/test/TEST-01-BASIC/test-init.sh . I can take this hunk out of this PR to have a separate discussion.